spec/cc/engine/analyzers/php/main_spec.rb

```diff
@@ -38,14 +38,14 @@
       expect(json["categories"]).to eq(["Duplication"])
       expect(json["location"]).to eq({
         "path" => "foo.php",
-        "lines" => { "begin" => 2, "end" => 6 },
+        "lines" => { "begin" => 2, "end" => 8 },
       })
-      expect(json["remediation_points"]).to eq(900_000)
+      expect(json["remediation_points"]).to eq(2_200_000)
```

Comment:
Should we adjust our remediation point multiplier as well so that the same duplication issues are given a similar remediation estimate?

Reply:
Ah, I didn't know there was a multiplier to adjust. I'll take a look at that. 👍

Reply:
Please see [here](https://github.com/codeclimate/codeclimate-duplication/pull/256#issuecomment-338010509) for response!